### PR TITLE
feat(Text): implement wholeText

### PR DESCRIPTION
**Pull request by bigopon**

### CODE REVIEW

**Comment by WebReflection** [comment]
Thanks @bigopon , I've discovered something new today !!!

However, this project will stay 100% code covered so without any test I won't merge it.

Why is that? For instance, you have two infinite loops and one typo in your PR, with a test you would have easily discovered both 😉 

Mind adding a test and fixing issues ? Thanks !

File: src/Text.js [hunk 1]
```diff
@@ -8,4 +8,21 @@ module.exports = class Text extends CharacterData {
     this.nodeType = Node.TEXT_NODE;
     this.nodeName = '#text';
   }
+
+  get wholeText() {
+    let text = this.textContent;
+    let prev = this.previousSibling;
+    while (prev && prev.nodeType === 3) {
```
Comment (WebReflection) [inline src/Text.js]: infinite loop, you never reassign `prev`

File: src/Text.js [hunk 2]
```diff
@@ -8,4 +8,21 @@ module.exports = class Text extends CharacterData {
...
+      text = prev.textContent + text;
+    }
+    let next = this.nextSibling;
+    while (next && next.nodeType === 3) {
```
Comment (WebReflection) [inline src/Text.js]: infinite loop, you never reassign `next`

File: src/Text.js [hunk 3]
```diff
@@ -8,4 +8,21 @@ module.exports = class Text extends CharacterData {
...
+      text = text + nexzt.textContent;
```
Comment (WebReflection) [inline src/Text.js]: `nexzt` is a typo

**Comment by bigopon** [comment]
@WebReflection done. Was unable to get branches to 100% though. I'm also not sure about the coverage report so I excluded it in fix commit. moral of the story: run test when PR 😄

**Pull request merged**